airflow/kubernetes/pod_generator.py

```diff
@@ -224,6 +228,8 @@ def __init__(  # pylint: disable=too-many-arguments,too-many-locals
         else:
             self.ud_pod = pod
 
+        self.docker_repository = docker_repository
```

Comment:
Would like to update this field name to be more specific to xcom-sidecar

Reply:
Ok! this match my previous comment! ;)